lib/web_ui/test/canvaskit/flutter_tester_emulation_golden_test.dart

```diff
@@ -0,0 +1,118 @@
+        () async {
+      final CkPictureRecorder recorder = CkPictureRecorder();
+      final CkCanvas canvas = recorder.beginRecording(kDefaultRegion);
+      canvas.translate(10, 10);
+
+      void _drawTextWithOutline(String text, {
+        String? paragraphFontFamily,
+        String? textFontFamily,
+        List<String>? textFontFallbacks,
+        String? strutStyleFontFamily,
+        List<String>? strutStyleFontFallbacks,
+        double? strutStyleFontSize,
+      }) {
+        final CkStrutStyle? strutStyle;
+
+        if (strutStyleFontFamily != null || strutStyleFontFallbacks != null || strutStyleFontSize != null) {
+          strutStyle = CkStrutStyle(
+            fontFamily: strutStyleFontFamily,
+            fontFamilyFallback: strutStyleFontFallbacks,
+            fontSize: strutStyleFontSize,
+          );
+        } else {
+          strutStyle = null;
+        }
+
+        final CkParagraphBuilder builder = CkParagraphBuilder(CkParagraphStyle(
+          fontFamily: paragraphFontFamily,
+          strutStyle: strutStyle,
+        ));
+
+        final bool needsTextStyle = textFontFamily != null || textFontFallbacks != null;
+
+        if (needsTextStyle) {
+          builder.pushStyle(CkTextStyle(
+            fontFamily: textFontFamily,
+            fontFamilyFallback: textFontFallbacks,
+          ));
+        }
+
+        builder.addText(text);
+
+        if (needsTextStyle) {
+          builder.pop();
+        }
+
+        final CkParagraph paragraph = builder.build();
+        paragraph.layout(const ui.ParagraphConstraints(width: 10000));
+        canvas.drawParagraph(paragraph, ui.Offset.zero);
+        canvas.drawRect(
+          ui.Rect.fromLTWH(-4, -4, paragraph.maxIntrinsicWidth + 8, paragraph.height + 8),
+          CkPaint()
+            ..style = ui.PaintingStyle.stroke
+            ..strokeWidth = 1,
+        );
+        canvas.translate(
+          0,
+          paragraph.height + 16,
+        );
+      }
+
+      _drawTextWithOutline('default');
+      _drawTextWithOutline(
+        'roboto paragraph',
+        paragraphFontFamily: 'Roboto',
+      );
+      _drawTextWithOutline(
+        'roboto text',
+        textFontFamily: 'Roboto',
+      );
+      _drawTextWithOutline(
+        'roboto text fallback',
+        textFontFallbacks: <String>['Roboto'],
+      );
+      _drawTextWithOutline(
+        'roboto strut style',
+        strutStyleFontFamily: 'Roboto',
+        strutStyleFontSize: 40,
+      );
+      _drawTextWithOutline(
+        'roboto strut style fallback',
+        strutStyleFontFallbacks: <String>['Roboto'],
+        strutStyleFontSize: 40,
+      );
+
+      await matchPictureGolden(
+        'canvaskit_defaults_to_ahem.png',
+        recorder.endRecording(),
+        region: kDefaultRegion,
+      );
+    });
+    // TODO(yjbanov): https://github.com/flutter/flutter/issues/60040
+    // TODO(yjbanov): https://github.com/flutter/flutter/issues/71520
```

Comment:
Do we even run canvaskit tests on other browsers?

Reply:
Yes, we run in Chrome, Safari desktop, and Firefox, but we screenshot only on Chrome.

Reply:
Ah okay that makes sense cuz I only saw Canvaskit/Chrome screenshots in Gold.

Reply:
We can take screenshots from Safari for iOS already. The only thing that's stopping us is that LUCI needs to upgrade the iOS Simulator to 14 or newer. We're still testing on iOS 13.

Reply:
(iOS Simultaor 14 added support for WebAssembly, which is what's actually blocking)